hs_core/page_processors.py

```diff
@@ -49,7 +49,7 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
     content_model = page.get_content_model()
     discoverable = content_model.raccess.discoverable
     validation_error = None
-
+    content_model.is_mine = content_model.rlabels.is_mine(user)
```

Comment:
Suggestion: For the future, it is rather important for a user to know whether a resource in the my resources page is public. Right now, only public and discoverable resources *can* be tagged with "mine". This, however, is not a limit of the API. It is probably a better idea in the future to depict whether a resource is public (which of course means that it is in fact in the is_mine list or it would not appear here) than to distinguish based upon is_mine and not is_mine. Just food for thought.